packages/plugin/webpack/src/WebpackConfig.ts

```diff
@@ -194,25 +194,32 @@ export default class WebpackConfigGenerator {
           target: this.rendererTarget(entryPoint),
           devtool: this.rendererSourceMapOption,
           mode: this.mode,
-          output: {
+          output: Object.assign(entryPoint.output || {}, {
```

Comment:
This is a mutation of the provided `output` object and isn't what you want to be doing.  Please use the spread operator.

```suggestion
          output: {
            ...(entryPoint.output || {}),
```

Reply:
good catch, will fix